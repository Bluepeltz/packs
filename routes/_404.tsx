export default () => {
  return (
    <h1 style={{ letterSpacing: '1em' }}>
      404
    </h1>
  );
};
