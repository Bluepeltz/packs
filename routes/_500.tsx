export default () => {
  return (
    <h1 style={{ letterSpacing: '1em' }}>
      OOPS
    </h1>
  );
};
