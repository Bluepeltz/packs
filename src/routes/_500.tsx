export default () => {
  return (
    <h1
      style={{
        margin: '0 auto',
        letterSpacing: '1em',
      }}
    >
      OOPS
    </h1>
  );
};
