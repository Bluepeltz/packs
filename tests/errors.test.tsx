import { renderToString } from 'preact-render-to-string';

import { assertSnapshot } from '$std/testing/snapshot.ts';

import _404 from '../routes/_404.tsx';
import _500 from '../routes/_500.tsx';

Deno.test('/404', async (test) => {
  await assertSnapshot(
    test,
    renderToString(_404()),
  );
});

Deno.test('/500', async (test) => {
  await assertSnapshot(
    test,
    renderToString(_500()),
  );
});
