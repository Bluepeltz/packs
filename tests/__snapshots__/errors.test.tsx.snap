export const snapshot = {};

snapshot[`/404 1`] = `'<h1 style="letter-spacing:1em;">404</h1>'`;

snapshot[`/500 1`] = `'<h1 style="letter-spacing:1em;">OOPS</h1>'`;
